packages/compass-aggregations/src/plugin.jsx

```diff
@@ -27,13 +28,15 @@ class Plugin extends Component {
    */
   render() {
     return (
-      <Provider store={this.props.store}>
-        <Aggregations
-          showExportButton={this.props.showExportButton}
-          showRunButton={this.props.showRunButton}
-          showExplainButton={this.props.showExplainButton}
-        />
-      </Provider>
+      <ConfirmationModalArea>
```

Comment:
why should this not be global?

Reply:
Because when plugin is rendered outside of compass (DE), we still want the confirmation modal to show up, same (but less critical) issue affects the global toast we have, also mentioned in https://github.com/mongodb-js/compass/pull/4079#discussion_r1113059159

Reply:
This is also a no-op when used inside compass: https://github.com/mongodb-js/compass/blob/7d65b08a1e2d2360b97391ed11253aa156c8bb1e/packages/compass-components/src/hooks/use-confirmation.tsx#L124-L126

Reply:
Ahhh so it is global, but is also here, got it!

Reply:
Should we remember to clean this up or leave a comment?